src/session.js

```diff
@@ -219,6 +219,8 @@ export default class Session {
       let conn = this.connections.get(util.connKey(localClientID, remoteClientID));
       if (conn) {
         conn.sendAck(packet.getSequenceId());
+      } else {
+        throw new errors.ConnNotFoundError(util.connKey(localClientID, remoteClientID));
```

Comment:
The error message here contains only connKey. It's better to be more verbose, like `Conn xxx not found`

Reply:
Yes, modified.